addons/account_payment/views/account_portal_templates.xml

```diff
@@ -1,6 +1,6 @@
 <odoo>
     <template id="portal_my_invoices_payment" name="Payment on My Invoices" inherit_id="account.portal_my_invoices">
-        <xpath expr="//t[@t-call='portal.portal_table']/thead/tr/th[last()]" position="before">
+        <xpath expr="//t[@t-call='portal.portal_table']//th[@name='amount_due']" position="before">
```

Comment:
I left the commit adding the names in this PR and moved the other commit to a PR for master: https://github.com/odoo/odoo/pull/183860

Reply:
@CLaurelB Rather move both to master (on a branch with the same name, so that runbot synchronizes it with the enterprise counterpart). Merging this in 17.0 already has no real value ;  and is error-prone. Imagine some bugfix or new module introduced in 17.0 wants to override the view. The dev will see the names and use them. But actually, those names won't be set in every db, since they require a module update => this will break.

Reply:
Hi @oco-odoo,

> Merging this in 17.0 already has no real value

I consider this actually adds value. The problem is, these elements can't be reached on a deterministic manner, which is why we need this change. This is an usually accepted change in stable, a few examples would be:
- https://github.com/odoo/odoo/pull/59453
- https://github.com/odoo/odoo/pull/37769

> so that runbot synchronizes it with the enterprise counterpart

In this case, there's no enterprise PR involved.

> Imagine some bugfix or new module introduced in 17.0 wants to override the view. The dev will see the names and use them. But actually, those names won't be set in every db, since they require a module update

I think, in such a case, the dev won't be able to reach elements using deterministic xpaths anyway. Which, depending on the PoV, could be even worse because the new module could inadvertently produce incorrect or undesired behaviors.

What do you think?

Regards,